back-end/src/routes/api.js

```diff
@@ -104,8 +97,203 @@ const uploadFile = (request, response) => {
   });
 };
 
-//uploadFileAsync(process.argv[2])
+/**
+ * Retrieve all photos uploaded by user.
+ * @param {email: String} request 
+ * @param {[{photo: Object, desc: String, uploadDate: String, title: String, likes: String}]} response 
+ */
+const getPhotoSocial = (request, response) => {
+  (async () => {
+    var statusCode = 400;
+    var message = "User does not exist";
+    console.debug(request.headers.email);
```

Comment:
why headers?

Reply:
Nothing in request.body was getting read I assume since these are get requests. If I changed them to put requests then body should be fine.

Reply:
UPDATE: nvm I'm an idiot. Had to configure Postman differently.